i8_terminal/types/period_param_type.py

```diff
@@ -64,8 +39,8 @@ class PeriodParamType(AutoCompleteChoice):
 
     def get_suggestions(self, keyword: str, pre_populate: bool = True, metric: str = None) -> List[Tuple[str, str]]:
         if not self.is_loaded:
-            self.metrics = get_all_metrics_types_dict()
-        self.set_choices(PERIODS.get(self.metrics[metric if metric else ""], []))
+            self.metrics = get_all_metrics_default_period_types_dict()
+        self.set_choices(PERIODS.get(self.metrics[metric if metric else ""], [("p", "Default period type (NA)")]))
```

Comment:
Since we only have period types for financial metrics, make it simpler. No dictionary is needed. Something like

```
if defatul_pt in ["Q", "FY", "TTM", "YTD"]:
    self.set_choices(....
else:
    self.set_choices([("p", "Default period type (NA)")])
```